lib/datadog/core/configuration/settings.rb

```diff
@@ -663,6 +663,14 @@ def initialize(*_)
             o.type :bool
           end
 
+          # Enable agentless mode for telemetry: submit telemetry events directly to the intake without Datadog Agent.
+          #
+          # @return [Boolean]
+          option :agentless_enabled do |o|
```

Comment:
This description is very internal-oriented, but this is a user-visible setting. Do we think customers will be enabling this themselves?

If so, I would suggest tweaking the comment to better explain why/when you'd want this. If we don't think they'll use this directly, perhaps add a "# We do not recommend tweaking this value." or something to make it clear that this setting is for "Datadog stuff" and usually you don't need to care about it.

Reply:
Oops, I forgot to mark it internal only. Customers are not supposed to do it by themselves.